### ISSUE TITLE
feature request: pluscal/tla cheatsheet

### ISSUE THREAD

**Issue by acud** [hwayne/learntla-v2#68]
While going through the website and trying to write some specs, it was noticeable that there's a certain lack of a page that puts it all together in a way that one can easily refer to and see the big picture. 

As an example I can give [this](https://cheats.rs/), a really nice Rust cheatsheet page.

Due to the semantic differences in basic operators between PlusCal/TLA and conventional languages, I think this could be something that would help people to more easily onboard.

It is, however, an undertaking, but it would definitely be useful.

The `Standard Modules` page exists, and it could be repurposed to do this. It appears that the current purpose of the page is to serve as a "standard library" equivalent for Pluscal/TLA. However, given that the standard library is pretty mature (and lean), it might be more informative and helpful to just "spec it all out" with the stuff that's been left out - the actual nuts and bolts that are given in the guide (from the most basic things like operators, conditionals, logical operators, etc). I find myself running through the entire guide trying to figure out how to do very basic things still, and it's definitely a time consumer since one needs to keep a mind-map of where a given concept has been introduced. This is especially noticeable in the first few chapters where all the basics are spread out.

Would be interested to hear what you think of this.

**Comment by hwayne** [comment]
I agree, this would be really helpful. It's just a lot of work to implement,  so it might take some time. I'd prefer to do it as a separate page from the standard modules.

What do you think of the legacy learntla reference pages? Would you be willing to try them for a few days and tell me the point points on those? 

* https://old.learntla.com/reference/types/
* https://old.learntla.com/reference/structure/
* https://old.learntla.com/reference/expressions/

**Comment by acud** [comment]
Sure :+1:

**Comment by fgmarand** [comment]
FWIW there is one at https://ocamlpro.com/assets/pdf/tla-cheat-sheet-v1.pdf : maybe just link to it ?
